### ISSUE TITLE
Allow saving bound BTC addresses

### ISSUE THREAD

**Issue by montycrypto** [skycoin/skycoin#510]
For the OTC sales I need an overview of requested bound BTC addresses. 

I could handle the teller calls in the front-end, in this case I would simply need GET/POST to read and add BTC addresses. We could also let our back-end handle it, but for now I prefer the front-end.

Until this API is completed we can use localStorage, this does not work perfectly within Electron though - which is undesirable for such important logic.

**Comment by gz-c** [comment]
@senyoret1 does the wallet still have the "teller"/"otc" integration code in it?

**Comment by gz-c** [comment]
Teller integration is not a priority; if the code has been removed we'll close this issue.

If the code is still there, we can keep this open, but we have no intention to run teller again for ourselves. However it is part of the fiber package, so it's something we may want.

This depends on #2119

**Comment by gz-c** [comment]
The teller integration code is out of date and if/when we decide to continue with it we can start over and include this as part of the project
